sdks/python/setup.py

```diff
@@ -98,7 +98,7 @@ def get_version():
     'avro>=1.8.1,<2.0.0',
     'crcmod>=1.7,<2.0',
     'dill==0.2.6',
-    'grpcio>=1.0,<2',
```

Comment:
What is the reason for this change? AFAIK, some runner (Dataflow) requires grpcio 1.3? Also, it is became more restrictive for users each time we reduce the list of allowed versions for a dependency.

Reply:
grpcio 1.8 is the first python version which allows passing/receiving client headers

Reply:
Sounds good. Just try running dataflow runner test before finishing this PR.

Reply:
Sure, Tried running apache_beam.runners.dataflow.dataflow_runner_test.DataflowRunnerTest

Reply:
I meant running a job on dataflow using: https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_beam_PostCommit_Python_Verify.groovy

Commenting "Run Python PostCommit" on the PR should trigger one.